docs/metrics/diarization-error.md

```diff
@@ -0,0 +1,103 @@
+
+### Definition
+
+Formally, DER is defined as the duration of false alarm, missed detection, and speaker confusion errors divided by the reference duration.
+
+$$
+\text{DER} = \frac{\text{false alarm} + \text{missed detection} + \text{speaker confusion}}{\text{reference duration}}
+$$
+
+### Example
+Using the following time segments, let's calculate the DER of the candidate text.
+
+```python
+from pyannote.core import Segment, Annotation
+
+reference = Annotation()
+reference[Segment(0, 5)] = 'C'
+reference[Segment(5, 9)] = 'D'
+reference[Segment(10, 14)] = 'A'
+reference[Segment(14, 15)] = 'D'
+reference[Segment(17, 20)] = 'C'
+reference[Segment(22, 25)] = 'B'
+
+hypothesis = Annotation()
+hypothesis[Segment(0, 8)] = 'C'
+hypothesis[Segment(11, 15)] = 'A'
+hypothesis[Segment(17, 21)] = 'C'
+hypothesis[Segment(23, 25)] = 'B'
+```
+
+![Visualization](../assets/images/diarization2.png)
+
+??? example "Step 1. Calculate Error Durations"
+    <b>False Alarm</b> <br>
+    Our candidate diarization contains speech that doesn't exist in the reference diarization at time segment `[20, 21]`. Our false alarm duration is 1 second.
+
+    <b>Missed Detection</b><br>
+    Our model fails to detect speech at time segments `[8, 9]`, `[10, 11]` and `[22, 23]`. Our missed detection duration is 3 seconds.
+
+    <b>Speaker Confusion</b><br>
+    Our candidate misidentifies speaker `D` as `C` at time segment `[5, 8]` and misidentifies speaker `D` as `A` in segment `[14, 15]`. Our speaker confusion duration is 4 seconds.
+
+??? example "Step 2. Calculate Diarization Error Rate"
+    Using the formula, we get:
+
+    $$
+    \begin{align*}
+    \text{DER} &= \frac{\text{false alarm} + \text{missed detection} + \text{speaker confusion}}{\text{reference duration}} \\
+               &= \frac{1 + 3 + 4}{20} \\
+               &= 0.4
+    \end{align*}
+    $$
+
+    Our diarization error rate is 0.4.
+
+## Limitiations and Biases
+
+Though DER provides a strong insight into the accuracy of speaker labels and predicted segments, it fails to pinpoint the specific components of a speaker diarization system that may cause it to perform poorly. As such, it is a good metric to evaluate the overall performance of a diarization system, but should be used alongside targeted metrics such as [detection error rate](https://pyannote.github.io/pyannote-metrics/reference.html) and [segmentation coverage](https://pyannote.github.io/pyannote-metrics/reference.html) to evaluate the individual models that detect voice activity, or speaker changes. Furthermore, diarization error rate does not place any weight on the accuracy of generated transcript texts, as it is used solely to measure the error in a diarization system.
```

Comment:
Let's use the links that can direct us to the exact heading instead.  It would be nice to have metrics docs for these metrics as well and link our own docs. 
https://pyannote.github.io/pyannote-metrics/reference.html#detection
https://pyannote.github.io/pyannote-metrics/reference.html#segmentation